### PR TITLE
[REF] [Import] remove pointless fini functions.

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
These never do anything - there are there only through copy & paste + a the abstract class requiring them.

In fact all the abstract functions are dumb - the are only called internally
within the classes so I have gotten rid of the abstractness so they
can be removed at will. import makes sense - but the signature doesn't...

Overview
----------------------------------------
_A brief description of the pull request. Keep technical jargon to a minimum. Hyperlink relevant discussions._

Before
----------------------------------------
_What is the old user-interface or technical-contract (as appropriate)?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

After
----------------------------------------
_What changed? What is new old user-interface or technical-contract?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

Technical Details
----------------------------------------
_If the PR involves technical details/changes/considerations which would not be manifest to a casual developer skimming the above sections, please describe the details here._

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

File: CRM/Import/Parser.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * @package CRM
  * @copyright CiviCRM LLC https://civicrm.org/licensing
  */
-abstract class CRM_Import_Parser {
+class CRM_Import_Parser {
```
Comment (colemanw) [inline CRM/Import/Parser.php:17]: I'm not sure I see the reason for removing `abstract`. Is there a reason this class would be used directly?

Comment (eileenmcnaughton) [reply CRM/Import/Parser.php:17]: @colemanw - yeah it's only that I removed the last abstract funciton - but I can put it back - hopefully at some point we will come up with a better interface & migrate the classes over...

Comment (colemanw) [reply CRM/Import/Parser.php:17]: Ok. I'd say let's keep it `abstract` for now.

Comment (eileenmcnaughton) [reply CRM/Import/Parser.php]: @colemanw ok - I also removed the change to the Custom partser class so this won't conflict with https://github.com/civicrm/civicrm-core/pull/23222 (I'll just have to follow up once both are merged)

**Pull request merged**